app/models/micropost.rb

```diff
@@ -0,0 +1,20 @@
+class Micropost < ApplicationRecord
+  belongs_to :user
+  has_one_attached :image
+  default_scope ->{order(created_at: :desc)}
```

Comment:
không dùng default scope em tìm hiểu lý do tại sao nhé

Reply:
Theo em tìm hiểu thì do default_scope làm ảnh hưởng đến câu truy vấn và đè lên scope thường.
Dùng scope giúp ghép các câu truy vấn nhỏ thành câu truy vấn lớn, phức tạp và còn áp dụng được với toán tử nữa